Speech/src/V2/Client/SpeechClient.php

```diff
@@ -185,6 +185,25 @@ public function resumeOperation($operationName, $methodName = null)
         return $operation;
     }
 
+    /**
+     * Create the default operation client for the service.
+     *
+     * @param array $options ClientOptions for the client.
+     *
+     * @return OperationsClient
+     */
+    private function createOperationsClient(array $options)
+    {
+        // Unset client-specific configuration options
+        unset($options['serviceName'], $options['clientConfig'], $options['descriptorsConfigPath']);
+
+        if (isset($options['operationsClient'])) {
+            return $options['operationsClient'];
+        }
+
+        return new OperationsClient($options);
+    }
```

Comment:
Even though the breaking change detector isn't detecting it, this IS a breaking change - the current clients are returning a different class for `createOperationsClient`, which is exposed to the customer in `getOperationsClient`.